components/MachineCard/index.js

```diff
@@ -49,12 +49,14 @@ export default function MachineCard({ machine }) {
         {" "}
         <BsFillGearFill />
       </EditButton>
-      <StyledImage
-        src={machine.machineImage}
-        alt="No image found"
-        width={200}
-        height={200}
-      />
+      {machine.machineImage && (
+        <StyledImage
+          src={machine.machineImage}
+          alt="No image found"
```

Comment:
I think in this case there will be an image found (due to `machine.machineImage &&`

does the `machine` object have a property on it you could use for the `alt`?

Reply:
i added that line incase the user didnt upload an image because he didnt need it or forgot to.
in those cases the card would render the round box shadow where the image would be.
i think there was a text "no image found" rendered before but it wasnt centered and i thought it to looked better this way.
hope i managed to explain it somehow understandable🫥